src/libraries/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -597,4 +597,7 @@
   <data name="PlatformNotSupported_NetHttp" xml:space="preserve">
     <value>System.Net.Http is not supported on this platform.</value>
   </data>
-</root>
+  <data name="net_http_chunk_envelope_too_large" xml:space="preserve">
+    <value>The HTTP/1.1 response chunk envelope was too large.</value>
```

Comment:
Do we use the term "envelope" elsewhere?

Reply:
Whatever wording we choose, let's just try to be consistent.  I don't have a strong preference about what exact words are chosen as long as it's clear and accurate.

Reply:
@dotnet/ncl any thoughts on the naming here? Are we okay with "chunk envelope"?

Reply:
I would personally stay with  "..response chunk was too large." e.g. not trying to match particular RFC or protocol version exactly.

Reply:
This would have shown up as "headers length exceeded" before, so at least mentioning the chunk may be a good enough improvement

Reply:
Changed it to avoid mentioning 'envelope'.